Tests/Serializer/BaseSerializerHandlerTest.php

```diff
@@ -69,6 +69,34 @@ public function testGetSubscribingMethods()
         $this->assertSame($methods, $expectedMethods);
     }
 
+    public function testSetFormats()
```

Comment:
IMO this test is testing the default value of array, may be I can change the name of method instead of deleting the whole method.